docs/userguide/active_learning.md

```diff
@@ -0,0 +1,68 @@
+# Active Learning
+When labeling unmeasured data points, e.g. for a data acquisition campaign to gather 
+data for a machine learning model, it can be beneficial to follow a guided approach 
+rather than randomly measuring data. If this is done via iteratively measuring points 
+according to a criterion proportional to the current model's uncertainty, the 
+method is called `active learning`.
+
+Active learning can be seen as a special case of Bayesian optimization. If we have the 
+above-mentioned criterion and set up a Bayesian optimization campaign to recommend 
+points with the highest uncertainty, we achieve active learning via Bayesian 
+optimization. In practice, we can leverage the fact that we have a posterior 
+distribution, and compute an acquisition function that represents the overall 
+uncertainty about any given point in the searchspace.
+
+Below you find which acquisition functions in BayBE are suitable for this endeavor 
+including a few guidelines. Furthermore, BayBE offers you the ability to choose a 
+custom surrogate model. This means you can use the model you intend to measure points 
+for already during the data acquisition phase (more on this soon).
```

Comment:
Why "more on this soon"? We should either make this more specific or just don't mention this as this just shows that stuff is still missing, and people get 0 information on when they can expect more information if we write it like this.

Reply:
this is there to prepare a link to the userguide explaining custom models, but since tis missing I cant put it yet

the alternative would be not to mention it at all yet, let me know your pref

Reply:
I removed it as theres another reason: we should test it first before mentioning